frontend/server/src/Controllers/Run.php

```diff
@@ -5,7 +5,10 @@
 /**
  * RunController
  *
+ * @author joemmanuel
+ * @psalm-type ProblemCases=array<string, array<string, string>>
  * @psalm-type RunMetadata=array{verdict: string, time: float, sys_time: int, wall_time: float, memory: int}
+ * @psalm-type RunDetails=array{admin: bool, alias: string, cases?: ProblemCases|null, compile_error?: string, details?: array{compile_meta?: array<string, RunMetadata>, contest_score: float, groups?: list<array{cases: list<array{contest_score: float, max_score: float, meta: RunMetadata, name: string, score: float, verdict: string}>, contest_score: float, group: string, max_score: float, score: float, verdict?: string}>, judged_by: string, max_score?: float, memory?: float, score: float, time?: float, verdict: string, wall_time?: float}, guid: string, judged_by?: string, language: string, logs?: string, show_diff: string, source?: string}
```

Comment:
```suggestion
 * @psalm-type RunDetails=array{admin: bool, alias: string, cases?: ProblemCases, compile_error?: string, details?: array{compile_meta?: array<string, RunMetadata>, contest_score: float, groups?: list<array{cases: list<array{contest_score: float, max_score: float, meta: RunMetadata, name: string, score: float, verdict: string}>, contest_score: float, group: string, max_score: float, score: float, verdict?: string}>, judged_by: string, max_score?: float, memory?: float, score: float, time?: float, verdict: string, wall_time?: float}, guid: string, judged_by?: string, language: string, logs?: string, show_diff: string, source?: string}
```

bajo qué circunstancias se va a regresar `cases` como `null`?

Reply:
Ah, en ninguna. Se quita el `null`